actions/setup-yq/action.yml

```diff
@@ -0,0 +1,65 @@
+name: Install yq
+description: Sets up yq yaml parser - https://github.com/mikefarah/yq
+runs:
+  using: composite
+  steps:
+    - id: download-yq
+      shell: bash
+      run: |
+        set -e  # Exit on error
+        
+        # Configuration
+        ARCH="linux_amd64"
+        INSTALL_PATH="/usr/local/bin/yq"
+        
+        # Get latest version
+        VERSION=$(curl -sL https://api.github.com/repos/mikefarah/yq/releases/latest |
+        grep -o '"tag_name": *"[^"]*"' | cut -d'"' -f4 | sed 's/^v//')
+
+        echo "Latest version: $VERSION"
+      
+        # Create temporary directory and ensure cleanup with trap
+        TMP_DIR=$(mktemp -d)
+        trap "rm -rf '$TMP_DIR'" EXIT  # Auto-cleanup on exit
+        
+        cd "$TMP_DIR"
+        
+        # Download checksum files
+        BASE_URL="https://github.com/mikefarah/yq/releases/download/v${VERSION}"
```

Comment:
As in my earlier comment, why is the "v" prefix removed above when it's added back here?